### ISSUE TITLE
 Date and Time Error

### ISSUE THREAD

**Issue by hugopvl** [directus/directus#6818]
### Describe the Bug

When I select a date, July 15th, 2021 for example, after saving, the date automatically changes to July 13th, 2021, that is, they always change the date to the previous day.

When I edit an item, the update date is correct, but in the database it is minus 1 hour

![image](https://user-images.githubusercontent.com/81326296/125776684-a1cff586-7876-47ab-bc41-0db4454ccb54.png)
![image](https://user-images.githubusercontent.com/81326296/125776700-c526376e-2c00-47e2-ba27-a1be99c8b84b.png)

Type: DateTime
![image](https://user-images.githubusercontent.com/81326296/125776764-434602e5-6c04-4637-aef5-3574837cb193.png)

Time. SQLserver
![image](https://user-images.githubusercontent.com/81326296/125776831-933a4b8d-edfd-408e-9da6-ee2c2cda6512.png)

Select DataBAse: VS Directus
![image](https://user-images.githubusercontent.com/81326296/125776872-5644d472-8f1c-4566-8d94-1328536e7bdc.png)
![image](https://user-images.githubusercontent.com/81326296/125776914-e6213958-7fee-4c4c-bcdb-dfe6ea82a5f6.png)

Directus Server:
![image](https://user-images.githubusercontent.com/81326296/125776965-2b1c7bc8-85b4-4747-a054-5e8b4ef8945b.png)

Firefox Console:

![image](https://user-images.githubusercontent.com/81326296/125776995-5f1770b9-c876-4777-a145-1986f7f2022b.png)

### To Reproduce

I don't know how to reproduce this error, but I can say what I did differently, previously we had a MariaDB database, then I ran the npx directus ini command to create a new project and started using a database in SQLServer.
possibly that's when this started to happen, because in the previous project, it didn't change the dates, as the hours we never noticed.

### What version of Directus are you using?

9.0.0-rc.83

### What version of Node.js are you using?

V14.17.0

### What database are you using?

MSQLServer 2014 

### What browser are you using?

Chrome, Firefox

### What operating system are you using?

Cliente - Windows  Directus - Installed on Linux Server

### How are you deploying Directus?

Locally - Debian GNU/Linux 10 (buster)

**Comment by rijkvanzanten** [comment]
> previously we had a MariaDB database, then I ran the npx directus ini command to create a new project and started using a database in SQLServer.

Could be that SQL Server handles date conversion slightly differently, exposing this problem only on that DB

**Comment by hugopvl** [comment]
I've already had this in my mind, but then I went with the metabase to consult the database and it shows me the same date that is in the database, only in directus it is, 1 more hour.

![image](https://user-images.githubusercontent.com/81326296/125818196-707c911e-2959-4e2a-b063-def65a390ece.png)

![image](https://user-images.githubusercontent.com/81326296/125818244-e6162283-07fe-4eec-8bed-f4f55c771cc4.png)

![image](https://user-images.githubusercontent.com/81326296/125818521-c4cd810a-4bae-4ff4-ad23-c9aa35fff77d.png)

**Comment by hugopvl** [comment]
is it possible that directus is using universal time instead of local time? 
![image](https://user-images.githubusercontent.com/81326296/125819071-2982033f-a3a1-478d-b48e-90cd96e1ab3e.png)

**Comment by rijkvanzanten** [comment]
DateTime fields IO should be without timezone info, so should be rendered as-is.

**Comment by hugopvl** [comment]
Good morning, sorry to insist on this situation but see if i can help in this situation:

I created a new VM and made a connection to a DB in mysql, but I noticed that the directus date is still different from the local date:
![linux_myslq](https://user-images.githubusercontent.com/81326296/125944052-95b066fa-c371-4a65-a379-b43d108f8074.PNG)

then I compared the data in the database, and they are ok by the local date:

![linux_myslq2](https://user-images.githubusercontent.com/81326296/125944317-2481676d-af4f-4fb7-bec7-b96ca66fa364.PNG)

what is the difference between one installation and the other? the type of database, but does that make all the difference!
what is the difference between one installation and the other? the type of database, but does that make all the difference!

so I thought that the problem could be in the mysql connection, mssql sonnection, and I notice that in the mysql folder in node_modules, mysql makes the conversion from timezone to 'local': node_modules\mysql\lib\ConnectionConfig.js

`this.timezone           = options.timezone || 'local';`

In mssql there is no such configuration...

am I on the wrong track to discover this problem?

thank you very much for your attention
